tests/integration_test/run_int_test_from_pr.sh

```diff
@@ -31,37 +31,54 @@ else
 fi
 
 #Arguments to integration_test.sh:
-#ENV=$1
-#LIRA_MODE=$2
-#LIRA_VERSION=$3
-#PIPELINE_TOOLS_MODE=$4
-#PIPELINE_TOOLS_VERSION=$5
-#TENX_MODE=$6
-#TENX_VERSION=$7
-#SS2_MODE=$8
-#SS2_VERSION=$9
-#TENX_SUB_ID=${10}
-#SS2_SUB_ID=${11}
-#VAULT_TOKEN_PATH=${12}
-#SUBMIT_WDL_DIR=${13}
-#USE_CAAS=${14}
-#USE_HMAC=${15}
-#SUBMIT_AND_HOLD=${16}
+#ENV=${1}
+#LIRA_MODE=${2}
+#LIRA_VERSION=${3}
+#LIRA_DIR=${4}
+#SECONDARY_ANALYSIS_MODE=${5}
+#SECONDARY_ANALYSIS_VERSION=${6}
+#SECONDARY_ANALYSIS_DIR=${7}
+#PIPELINE_TOOLS_MODE=${8}
+#PIPELINE_TOOLS_VERSION=${9}
+#PIPELINE_TOOLS_DIR=${10}
+#TENX_MODE=${11}
+#TENX_VERSION=${12}
+#TENX_DIR=${13}
+#SS2_MODE=${14}
+#SS2_VERSION=${15}
+#SS2_DIR=${16}
+#TENX_SUB_ID=${17}
+#SS2_SUB_ID=${18}
+#VAULT_TOKEN_PATH=${19}
+#SUBMIT_WDL_DIR=${20}
+#USE_CAAS=${21}
+#USE_HMAC=${22}
+#SUBMIT_AND_HOLD=${23}
+#COLLECTION_NAME=${24:-"lira-${LIRA_ENVIRONMENT}"}
```

Comment:
There's no `LIRA_ENVIRONMENT` defined in this script

Reply:
It was referred to as "ENV" here. I updated it so that it would be referred to by the same value everywhere